bftengine/src/bcstatetransfer/AsyncStateTransferCRE.cpp

```diff
@@ -30,7 +30,8 @@ class Communication : public ICommunication {
   Communication(std::shared_ptr<MsgsCommunicator> msgsCommunicator, std::shared_ptr<MsgHandlersRegistrator> msgHandlers)
       : msgsCommunicator_{msgsCommunicator}, repId_{bftEngine::ReplicaConfig::instance().replicaId} {
     msgHandlers->registerMsgHandler(MsgCode::ClientReply, [&](bftEngine::impl::MessageBase* message) {
-      if (receiver_) receiver_->onNewMessage(message->senderId(), message->body(), message->size());
+      if (receiver_)
+        receiver_->onNewMessage(message->senderId(), message->body(), message->size(), message->senderId());
```

Comment:
No need to pass the same value twice - let the default value take place. Everywhere in this file.

Reply:
done.